pages/How-to/guide/linking-issue-with-pr.md

```diff
@@ -1,4 +1,48 @@
-<div align="center">
-  <img src="https://user-images.githubusercontent.com/74750414/169188784-680014eb-ae5a-4062-b9c0-40276f28a261.png" />
-</div>
+# How to Link and Issue no with a PR ❓
+### Things to keep in 🧠
+> You can link up to **ten issues** to each pull request.
 
+> The issue and pull request must be in the **same repository**.
+
+## Linking an issue with a PR 🔗 
+1. On GitHub.com, navigate to the main page of the repository.
+2. Under your repository name, click **Pull requests**.
+3. In the list of pull requests, click the pull request that you'd like to link to an issue.

```

Comment:
```suggestion
1. Open the PR in which you want to link the issue. ( Note: If you are a contributor you can only link an issue to your PR. Only maintainers can do with any PR.)
```

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix